WikiWatershed/model-my-watershed
# Add Shippensburg Future Land Use layers

## Overview

Adds three Shippensburg visual layers. See https://github.com/azavea/civic-apps-etl/pull/6 for the ETL stuff.

Connects #2810 

### Demo

![2018-06-22 18 27 47](https://user-images.githubusercontent.com/1430060/41801710-164201b0-764a-11e8-9591-94e4f1f64ef8.gif)

Also tagging @ajrobbins for visual review.

### Notes

There may be updates to the exact naming / legend / descriptions in the future.

## Testing Instructions

* Check out this branch and `bundle --debug`
* Go to [:8000/](http://localhost:8000/) and in the layer selector select the "Coverage" tab, scroll all the way down, and test the three Future Land Use layers
